v0/webhook/transform.js

```diff
@@ -0,0 +1,35 @@
+const {
+  defaultPostRequestConfig,
+  defaultRequestConfig,
+  isURL
+} = require("../util");
+
+function process(event) {
+  const { message, destination } = event;
+  const response = defaultRequestConfig();
+  if (destination.Config.webhookUrl && isURL(destination.Config.webhookUrl)) {
```

Comment:
Regex checks are generally costly. Let's move this check to config-backend/webapp and assume this is a valid URL.

Reply:
Removed the regex check